internals/processes.py

```diff
@@ -265,9 +263,18 @@ def process_to_dict(process):
                PI_EXPLAINER.name, PI_SPEC_LINK.name,
                PI_EST_TARGET_MILESTONE.name])],
       [approval_defs.ExperimentApproval],
-      core_enums.INTENT_IMPLEMENT_SHIP, core_enums.INTENT_EXTEND_TRIAL,
+      core_enums.INTENT_IMPLEMENT_SHIP, core_enums.INTENT_ORIGIN_TRIAL,
       stage_type=core_enums.STAGE_BLINK_ORIGIN_TRIAL),
 
+  ProcessStage(
```

Comment:
Since the Guide UX is driven by these process definitions, I think that every feature's Guide UX page will immediately show a stage to extend the origin trial.  I suppose that is OK.

Reply:
When we send this over to the front end, the extension stages are organized a little differently (as properties of the origin trial stage), so it seems like this is not the case. 🙂 No regression there as far as I can see.